services/motion/builtin/builtin_utils_test.go

```diff
@@ -0,0 +1,270 @@
+package builtin
```

Comment:
our `builtin_test.go` is getting rather large!
I pulled these out into a separate file for better organization.